lib/presentation/home_page/components/main_page_weather_content.dart

```diff
@@ -69,12 +69,14 @@ class _MainPageWeatherContentState extends State<MainPageWeatherContent> {
                   BuildContext context,
                   AsyncSnapshot snapshot,
                 ) {
+                  Widget child;
                   if (snapshot.hasData) {
                     dynamic data = snapshot.data;
                     double windConverter =
                         ((data.currentWeatherModel.windSpeed * 1 / 1000) /
                             (1 / 3600));
-                    return Container(
+                    child = Container(
+                      key: const ValueKey(1),
```

Comment:
Try without that and without key in Lottie asset, these are different types so it should works without using keys.

Reply:
And we don't need this keys as well.  It works also without this keys